ddl/partition.go

```diff
@@ -2039,6 +2039,10 @@ func (w *worker) onExchangeTablePartition(d *ddlCtx, t *meta.Meta, job *model.Jo
 		return ver, errors.Trace(err)
 	}
 
+	d.mu.RLock()
+	d.mu.hook.OnWatched(d.ctx)
```

Comment:
This looks like the first time this call is used, so I think you can also update the comment for the OnWatched interface in ddl/callback.go with this as an example.

Also not sure this is matching the description of 'OnWatched is called after watching owner is completed'.

Reply:
Indeed, what do you think of adding a new callback for `exchange partition with table ddl test`？

Reply:
I think renaming this callback should work, since it is not actually used anywhere?
How would you describe the state when it is supposed to be called? I.e. what should the name of the callback be?
@bb7133 do you have any suggestions or thoughts about how to handle this DDL Callback/hook?

Reply:
Basically, I would prefer to avoid calling the hook in a function like this.

Reply:
failpoint is enough

Reply:
@ymkzpx You could use failpoint instead.

Reply:
good suggesttion！